tico/passes/decompose_group_norm.py

```diff
@@ -88,6 +88,25 @@ class DecomposeGroupNorm(PassBase):
     def __init__(self):
         super().__init__()
 
+    def _insert_norm(self, graph, tensor, eps):
+        """
+        Insert (tensor - mean) / sqrt(var + eps)) into the graph
+          and return the normalized tensor node.
+        """
```

Comment:
```suggestion
        """
        Insert below arithmetic into the graph
        (tensor - mean) / sqrt(var + eps))
        ~~~~~~~~~~~~~~~  ~~~~~~~~~~~~~~~~~
        ^                 ^
        denoted as `dev`  denoted as `inv_std`
          and return the normalized tensor node.
        """
```
(optional)
How about to enrich description..? I suggest this because it was bit confused to me what the `dev` means at the first glance

Reply:
Then, I just rename the variable next PR:) Thanks.